__tests__/app.test.js

```diff
@@ -29,6 +29,16 @@ describe("Error Handling", () => {
         expect(body.msg).toBe("No article found with that id!");
       });
   });
+  test("status 400: responds with an error message when trying to patch with a value that cant be used", () => {
+    const update = { inc_votes: "bad request" };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(update)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid value being patched!");
+      });
+  });
 });
```

Comment:
I would recommend keeping the error handling for endpoint test with the happy path testing to make it clearer. 

You have only added one test for the patch, can you think of any other errors to test for?